domains/POAS/events/priority-correspondence-sent/index.md

```diff
@@ -0,0 +1,31 @@
+---
+name: priority-correspondence-sent
+version: 0.0.1
+summary: |
+  An OPG user has sent a letter to the online donor to inform about a problem which needs attention.
+producers:
+  - opg.poas.sirius-proposed
+consumers:
+  - opg.poas.makeregister
+owners:
+  - vega
+  - mrlpa
+---
+
+## Context
+
+An OPG user has identified that there is a problem with the donor's application and has sent a letter to the donor informing about it.
+
+## Trigger
+
+Sirius sends the event when the letter informing about the problem has been successfully sent by GOV.UK Notify.
+
+## Effect
+
+The task list in MRLPA shows a link to view the problem.
```

Comment:
I don't think MRLPA can link to anything, it will just show something like "you will receive a letter"

Reply:
@gregtyler Done